homeassistant/components/sensor/temper.py

```diff
@@ -56,7 +56,8 @@ def unit_of_measurement(self):
     def update(self):
         """Retrieve latest state."""
         try:
-            self.current_value = self.temper_device.get_temperature()
+            format_str = 'fahrenheit' if self.temp_unit == '°F' else 'celsius'
```

Comment:
Please import constant `TEMP_FAHRENHEIT` from `homeassistant.const` and compare with that.